From 5c844ea8b58444a6a8d84f340cff0b3be2f6351c Mon Sep 17 00:00:00 2001
From: Vitaly Wool <vitaly.wool@konsulko.com>
Date: Fri, 17 Nov 2017 19:15:47 +0200
Subject: [PATCH 12/12] ns_parse: add _ns_flagdata definition

_ns_flagdata definition is missing in latest NDK headers but it's
simple enough to be added directly.
---
 gweb/ns_parse.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/gweb/ns_parse.c b/gweb/ns_parse.c
index c2e6c6b1..9596e1a1 100644
--- a/gweb/ns_parse.c
+++ b/gweb/ns_parse.c
@@ -40,6 +40,10 @@ static void	setsection(ns_msg *msg, ns_sect sect);
 
 /* Public. */
 
+#ifdef ANDROID
+struct _ns_flagdata { int mask, shift; };
+#endif /* ANDROID */
+
 /* These need to be in the same order as the nres.h:ns_flag enum. */
 const struct _ns_flagdata _ns_flagdata[16] = {
 	{ 0x8000, 15 },		/*%< qr. */
@@ -276,4 +280,3 @@ setsection(ns_msg *msg, ns_sect sect) {
 	}
 }
 
- 
-- 
2.11.0

